R/mle-finder.R

```diff
@@ -0,0 +1,36 @@
+MLE_pseudo.inverse <- function(design,outcome){
+  return(solve(t(design)%*%design,t(design)%*%outcome))
+}
+
+linear.loglik <- function(MLE,design,outcome,noise_var = 1){
```

Comment:
```suggestion
linear.loglik <- function(MLE, design, outcome, noise_var = 1){
```

Try to keep things neat like this